Guessing numbers.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+This is a search task in which participants guess numbers between 0-100 and the computer tells them how many point were obtained from the guess. The participants were told their guess and scores as well as the guesses and scores of their neighbors so that the participants could choose to imitate high-scoring guesses from their peers.
+
+# References
+Main paper: https://www.researchgate.net/publication/23196347_Propagation_of_Innovations_in_Networked_Groups
+
+# Stimuli
+## The visual components
+![guesses](images/guesses.png)
+
+## Materials for alternative versions of the experiment 
+>There were two fixed random orderings of 8 payout functions (including the functions used in Study 2) and network structures that were counterbalanced between groups.
+>
+>The positions of the maxima were different for each of the 8 problems
+>
+>The network structure for each problem was either full, lattic, small-world, or random
```

Comment:
I'm not sure this is an alternative version- wouldn't this mean people in the current experiment were in either of these treatment conditions?